src/convertToObject.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
-/**
+/*
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleUpgrated = sourceString
```

Comment:
u can return it immediately

```suggestion
  return sourceString
```